src/main/java/uk/gov/hmcts/divorce/divorcecase/model/Solicitor.java

```diff
@@ -94,4 +104,17 @@ public boolean hasOrgId() {
         }
         return false;
     }
+
+    public String getAddress() {
+        if (YesOrNo.YES == getAddressOverseas() || this.address == null || this.address.equals("")) {
```

Comment:
`this.address == null || this.address.equals("")` can be replaced with StringUtils.isEmpty(this.address)